source/_includes/head.html

```diff
@@ -0,0 +1,16 @@
+
+<head>
+  <meta charset="utf-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1">
+  {%- seo -%}
+  <link rel="stylesheet" href="{{ "/assets/main.css" | relative_url }}">
+  {%- feed_meta -%}
+  {%- if jekyll.environment == 'production' and site.google_analytics -%}
+    {%- include google-analytics.html -%}
+  {%- endif -%}
+
+  {%- include custom-head.html -%}
```

Comment:
This is fine for now, but I also think we can handle this using liquid/jekyll logic, `if homepage include custom-head else do nothing` e.g.

Reply:
This is the horrible bit I want to get rid of :-). Interestingly the default head for the minimal theme includes this custom-head.html file:

https://github.com/jekyll/minima/blob/master/_includes/head.html

So in theory I shouldn't need this head.html but it didn't work with only the custom-head.html file...